### PR TITLE
Added RHEL7 install for fiftyone-db

### PR DESCRIPTION
**Pull request by AmineDiro**
## Fix 
This patch fixes the install issue  #1097 when installing fiftyone on RHEL7.
The changes include adding download url option to the db package's `setup.py` for rhel7 distribution.
```python 
MONGODB_DOWNLOAD_URLS = {
    "linux-x86_64": "https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.4.2.tgz",
    "linux-aarch64": "https://fastdl.mongodb.org/linux/mongodb-linux-aarch64-ubuntu1804-4.4.2.tgz",
    "mac": "https://fastdl.mongodb.org/osx/mongodb-macos-x86_64-4.4.2.tgz",
    "win": "https://fastdl.mongodb.org/windows/mongodb-windows-x86_64-4.4.2.zip",
    "ubuntu1604": {
        "manylinux1_x86_64": "https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1604-4.4.2.tgz",
    },
    "debian9": {
        "manylinux1_x86_64": "https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-debian92-4.4.2.tgz",
    },
    "rhel7": {
        "manylinux1_x86_64": "https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-rhel70-4.4.2.tgz",
    },
}

```
## Tests

This patch was testing by creating a local wheel package add installing it using docker test-envs for Centos7. 

```
FROM centos:7

RUN yum install -y python3-devel python3-pip gcc

ENV PATH="/root/.local/bin:${PATH}"

RUN pip3 install --user --upgrade pip setuptools wheel

WORKDIR /test
```

Added a tab in `docs/getting-started/troubleshooting` for rhel7 install fix.

-   [ ] App: FiftyOne application changes
-   [x] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

File: docs/source/getting_started/troubleshooting.rst [hunk 1]
```diff
@@ -194,6 +194,12 @@ installation by adding `--force-reinstall` to the commands below.
 
       pip install fiftyone-db-debian9
 
+  .. tab:: RHEL 7
```
Comment (benjaminpkane) [inline docs/source/getting_started/troubleshooting.rst:197]:
```suggestion
  .. tab:: RHEL (CentOS) 7
```

**Review by benjaminpkane — CHANGES_REQUESTED** [review]
Awesome! I just published the [package](https://pypi.org/project/fiftyone-db-rhel7/).

```
pip install fiftyone-db-rhel7
```

To make this PR complete, can you just add the build and upload steps for rhel to our `build-db` workflow?


Essentially, just add this ad the [debian9 build wheel step]( [debian9 step](https://github.com/voxel51/fiftyone/blob/develop/.github/workflows/build-db.yml)#L64):
```
      - name: Build wheel (RHEL/CentOS 7 x86_64)
        env:
          FIFTYONE_DB_BUILD_LINUX_DISTRO: rhel7
        run: |
          cd package/db
          python setup.py bdist_wheel --plat-name linux-x86_64
```

And this after the [debia9 upload step](https://github.com/voxel51/fiftyone/blob/7cf2a3af0b5d11ed4cf7f699283c7d466c323146/.github/workflows/build-db.yml#L85):
```
      - name: Upload wheel (RHEL/CentOS 7)
        uses: actions/upload-artifact@v2
        with:
          name: wheel-rhel7
          path: package/db/dist/*rhel7*.whl
```

**Comment by AmineDiro** [comment]
Thanks for your guidance, i added the necessary steps to the build. Hope everything is fine.

I'm also working on a tutorial on Fiftyone with MLFlow integration, hope i can share it soon.

**Review by benjaminpkane — APPROVED** [review]

**Comment by brimoor** [comment]
> I'm also working on a tutorial on Fiftyone with MLFlow integration, hope i can share it soon.

@AmineDiro thanks for your contribution! The MLflow integration sounds amazing too 💯 💯 💯

**Pull request merged**